README.md

```diff
@@ -10,3 +10,15 @@ in a time series fashion to provide additional insights about their project.
 # Vizr Installation and Upgrades
 
 See [INSTALL.md](INSTALL.md).
+
+# Vizr Permissions
+
+By default, all users of a project with Vizr enabled may view charts.
```

Comment:
I would probably remove the phrase "by default", since there's no way to change the behavior.